sql/src/main/java/io/crate/execution/engine/collect/files/LineProcessor.java

```diff
@@ -32,6 +32,7 @@
 
     private LineContext lineContext = new LineContext();
     private LineParser lineParser = new LineParser();
+    private long currentLineNumber = 0;
```

Comment:
would it make sense to keep the state only in the `LineContext` instead of having it into two places? Could have a `reset` and `incLineNumber` or something like that.